polaris-react/src/components/Text/Text.scss

```diff
@@ -112,46 +91,26 @@
 .headingXl {
   font-size: var(--p-font-size-300);
   line-height: var(--p-font-line-height-3);
-  font-weight: var(--p-font-weight-semibold);
-
-  #{$se23} & {
-    font-weight: var(--p-font-weight-bold);
-  }
+  font-weight: var(--p-font-weight-bold);
 
   @media #{$p-breakpoints-md-up} {
     font-size: var(--p-font-size-400);
-    line-height: var(--p-font-line-height-4);
-
-    #{$se23} & {
-      line-height: var(--p-font-line-height-5);
-    }
+    line-height: var(--p-font-line-height-5);
   }
 }
 
 .heading2xl {
-  font-size: var(--p-font-size-300);
-  line-height: var(--p-font-line-height-3);
-  font-weight: var(--p-font-weight-semibold);
-
-  #{$se23} & {
-    font-size: var(--p-font-size-400);
-    line-height: var(--p-font-line-height-5);
-    font-weight: var(--p-font-weight-bold);
-  }
+  font-size: var(--p-font-size-400);
+  line-height: var(--p-font-line-height-5);
+  font-weight: var(--p-font-weight-bold);
 
   @media #{$p-breakpoints-md-up} {
     font-size: var(--p-font-size-500);
-    line-height: var(--p-font-line-height-5);
-
-    #{$se23} & {
-      // Specificity bump
-      font-size: var(--p-font-size-500);
-      line-height: var(--p-font-line-height-6);
-    }
+    line-height: var(--p-font-line-height-6);
   }
 }
 
-@mixin se23-heading3xl {
+@mixin _heading3xl {
```

Comment:
Keeping these mixins and replacing the se23- prefix with an _, as they appear to be used in multiple classes. I'm guessing this is because we're going to be getting rid of certain Text variants. (i.e. .headingXs and .headingSm are identical in se23)